### DISCUSSION TITLE
[Feature]: Running the script without root privileges

### DISCUSSION THREAD

**Discussion by madgagarin** [coollabsio/coolify#3145] [Feature Requests]
### Description

It would be great in v4 to be able to install coolify via ssh under any user.  It is not safe to keep ssh root open.  In v3 this works great.

### Minimal Reproduction (if possible, example repository)

-

### Version

v4

**Comment by hades200082** [comment]
100% agree. root user should not be required. Installing coolify using sudo should be possible, and from that coolify should create it's own user with the needed permissions, add itself to sudoers, etc.

**Comment by ronnostic** [comment]
I recently stumbled on this project and saw a [scary revelation](https://news.ycombinator.com/item?id=35531244) by a user on ycombinator that Coolify is doing a `-v /var/run/docker.sock:/var/run/docker.sock`. 

Is this still the case?

**Comment by NickWoodward** [comment]
> I recently stumbled on this project and saw a [scary revelation](https://news.ycombinator.com/item?id=35531244) by a user on ycombinator that Coolify is doing a `-v /var/run/docker.sock:/var/run/docker.sock`.
> 
> Is this still the case?

Any update on this?

**Comment by Sandlayth** [comment]
Is this somewhere going to be taken care of?
